src/components/auth/LoginForm.jsx

```diff
@@ -22,7 +22,6 @@ const LoginForm = () => {
     checkPolicy: true,
   };
   const loginBtnHandler = (e, user) => {
-    console.log(location, "location-login")
     e.preventDefault();
```

Comment:
if you do e.preventDef in call itslef no need to pass e here

Reply:
noted